walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+Developers can utilize it to build applications that will work independent of the network state.
+Showcase application contains integration with all offline capabilities offered by Data Sync Framework.
+In footer we should see current network state and number of offline changes we currently have.
+
+image::images/offline-footer.png[integration, role="integr8ly-img-responsive"]
+
+=== Steps
+
+. Let's open create task view and fill all task details
+. Please do not press create yet
+. To leverage offline capabilities we need to make sure that our application looses conectivity with our server. 
+On the Mobile phone this can be done by turning on airplane mode. 
+In Chrome browser you can use the Network tab from Chrome Developer Tools to simulate offline mode. (open them using F12, or using Command+Option+I on macOS)
+. Make your device go offline
+. Press `Create` button to create an new task
+. We should see task created and our `Offline Changes` button in footer containing now one change.
+. Let's edit task multiple times
+. Each edit will create new Task
+. We can review all offline items directly by clicking on `Offline Changes` button.
+. Please restore connectivity (depending on your device)
+. We should see all `Offline Changes` reflected back to server and appearing in second instance of the application.
+
+[time=6]
+== Conflict resolution functionalities
+
+Data Conflicts can happen when resource we modified was also modified on server.
+DataSync Framework enables multiple strategies to deal with conflicts. 
+By default Showcase will resolve conflicts by applying all server side changes on top of the client.
+In case of direct conflicts client field value will be used. 
+To simulate conflict we need to go offline and make edits in two instance of the application at the same time.
+
+=== Steps
+
+. Use one of the existing task or create new one that will be visible on two devices.
+. Go offline 
+. Edit Task on first instance
+. Go to second instance of the application
+. Edit Task on second instance
+. At this point we should have server side updated by second instance of the application
+and first application not knowing about that change because it went offline.
+. This operation will result in conflict. 
+. Once we make first client online again - data conflict will occur.
+We should see popup and actuall server side data being replicated back to the client (conflict was resolved automatically)
+
+. Use the “Manage Tasks” page in the showcase client app.
+. Try create one task in one device, and it should show up immediately in another.
+. Try bring one client offline (, perform some actions, bring it online and you should see the changes synced to another app straight away.
+. To create conflicts, try bringing both clients offline, and modify the same item. The last instance of app that is online will receive conflicts.
+
+[time=35]
+== [Optional] Check the Data Synchonization Audit Logs
```

Comment:
@aliok Do you think there will be some changes in procedure post tech preview here?

Reply:
@wtrocki What's written here is mostly generic info which shouldn't change.

Reply:
Thank you!